src/modules/components/proposal/proposalActions/proposalActionsList/proposalActionTokenMint/proposalActionTokenMint.tsx

```diff
@@ -1,11 +1,15 @@
+import { ChainEntityType, useBlockExplorer } from '../../../../../hooks';
 import { MemberDataListItemStructure } from '../../../../member';
 import type { IProposalActionTokenMintProps } from './proposalActionTokenMint.api';
 
 export const ProposalActionTokenMint: React.FC<IProposalActionTokenMintProps> = (props) => {
-    const { action } = props;
+    const { action, wagmiConfig, chainId } = props;
     const { tokenSymbol, receiver } = action;
     const { currentBalance, newBalance, address, name, avatarSrc } = receiver;
 
+    const { buildEntityUrl } = useBlockExplorer({ chains: wagmiConfig?.chains, chainId });
+    const receiverBlockExplorerLink = buildEntityUrl({ type: ChainEntityType.ADDRESS, id: address });
```

Comment:
Could you add a tests here and on the `ProposalActionChangeMembers` component for these new functionalities?